creator-node/test/contentBlacklist.test.js

```diff
@@ -0,0 +1,295 @@
+const assert = require('assert')
```

Comment:
if we can have a test case for `CIDIsInBlacklist` that would be awesome cause i think that's the one we use the most. this is a SUPER hack, but in your beforeEach, we can probably seed the db using michael's new sync and export test so the db has users and tracks and files. then blacklist something and make sure the redis set reflects it. https://github.com/AudiusProject/audius-protocol/commit/09ce76be40d1392cf1573295dc789f02643bda52#diff-397511abb68a4d07a85bd9b4b9cb6065R248.